app/assets/stylesheets/_main.scss

```diff
@@ -0,0 +1,43 @@
+.main {
+  margin: 46px auto 0;
+  width: 325px;
```

Comment:
What will happen for mobile users that have a width of `320px`? (that's the size of an iPhone 5)

Maybe we could just put a `max-width: 600px` on here instead?